### DISCUSSION TITLE
Managing Bluetooth MAC addresses

### DISCUSSION THREAD

**Discussion by brandondb1** [mKeRix/room-assistant#419] [Q&A]
Hi, just setting this up, and it seems really cool!  I've scavenged all my spare Pi's for nodes...
Question: For the MAC addresses of Classic Bluetooth devices - do they need to be entered into each local.yml file on each node, or do they share that information?
If yes, which I think is the case, it would be great to have some kind of central registry so you don't have to ssh into and edit multiple files...say you have guests for a week and want to amaze them with as little work as possible.

Thanks again for this great tool, I'm looking forward to getting it fully up and running!

**Comment by fanuch** [comment]
I'm also new .. like literally watching the Pi download new!
Have a look at [Ansible and the instructions from the project](https://www.room-assistant.io/guide/quickstart-ansible.html#requirements).

Ansible uses `ssh`, yes, but it will check and update the relevant files from a common config.

**Comment by brandondb1** [comment]
Ah, right you are!  Thanks for pointing that out, also thanks (@fanuch and everyone) for not pointing out my obvious failure to fully read the instructions!
